puppetlabs_spec_helper.gemspec

```
@@ -31,7 +31,7 @@ Gem::Specification.new do |spec|
   spec.add_runtime_dependency 'mocha', '~> 1.0'
   spec.add_runtime_dependency 'pathspec', '>= 0.2', '< 2.0.0'
   spec.add_runtime_dependency 'puppet-lint', '~> 4.0'
-  spec.add_runtime_dependency 'puppet-syntax', '~> 4.0'
+  spec.add_runtime_dependency 'puppet-syntax', '~> 4.1'
```

```suggestion
  spec.add_runtime_dependency 'puppet-syntax', '~> 4.0', '>= 4.1'
```

@bastelfreak how do you feel about this? Is this even necessary (i.e. will there be another 4.x release before 5?)